xla/stream_executor/rocm/BUILD

```diff
@@ -104,6 +104,7 @@ cc_library(
         ":rocm_platform_id",
         "@com_google_absl//absl/functional:any_invocable",
         "@com_google_absl//absl/strings",
+        "//xla/stream_executor:kernel",
```

Comment:
`kernel` will be an internal target with private visibility, it should be just `//xla/stream_executor` to bring all SE dependencies